modules/create-crio-default-capabilities.adoc

```diff
@@ -0,0 +1,42 @@
+// Module included in the following assemblies:
+//
+// * machine_configuration/machine-configs-custom.adoc
+
+:_mod-docs-content-type: CONCEPT
```

Comment:
A lot of this info is conceptual, but the title is a gerund and there are some steps at the end, so it has some procedure elements as well. Should this have been structured as a proc rather than a concept?

Reply:
That comes from [here](https://github.com/openshift/openshift-docs/pull/78951#discussion_r1681290569).